war/src/main/js/api/securityConfig.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Provides a wrapper to interact with the security configuration
+ */
+
+var jenkins = require('../util/jenkins');
+var jquery = require('jquery-detached');
+var wh = require('window-handle');
+
+// gets the window containing a form, taking in to account top-level iframes
+var getWindow = function($form) {
+	var $ = jquery.getJQuery();
+	var wnd = wh.getWindow();
+	$(top.document).find('iframe').each(function() {
+		var windowFrame = this.contentWindow;
+		var $f = $(this).contents().find('form');
+		if($f.length > 0 && $form[0] === $f[0]) {
+			wnd = windowFrame;
+		}
+	});
+	return wnd;
+};
```

Comment:
This seems to be in the wrong module. Seems like a more general utility i.e. nothing to do with "security".